apps/web/src/components/settings/data-form-fields.tsx

```diff
@@ -113,6 +124,16 @@ export default function DataFormFields({ form, disabled }: Props) {
                 render={({ field }) => (
                     <FormItem>
                         <FormLabel>Strompreis (in €/kWh)</FormLabel>
+                        <Popover>
+                            <PopoverTrigger>
+                                <Info className="ml-2 h-4 w-4" />
+                            </PopoverTrigger>
+                            <PopoverContent className="text-s">
+                                An Stromanbieter zu entrichtender Preis pro Kilowattstunde und enthält Grundpreis,
```

Comment:
Ich persönlich finde die Formulierung nicht so schön. Mein Vorschlag ist:
```suggestion
                                An den Stromanbieter zu entrichtender Preis pro Kilowattstunde, der den Grundpreis ,Arbeitspreis und Steuern enthält.
```